iset.mm

```diff
@@ -99758,6 +99758,64 @@ A Cauchy sequence (as defined here, which has a rate of convergence
     LXCSVPWEXESVGURVHVPVRWAVPVQCDZVRNDABVIZVQPQVPVSVTWTXAUMVPXJTVRIJXKVQVJQVPVS
     VTWTXAVPVNTVSIJWOAVJQVPVOTVTIJWPBVJQVKVLVM $.
 
+  $( An upper bound for ` { A , B } ` .
+     (Contributed by Jim Kingdon, 20-Dec-2021.) $)
+  maxabsle $p |- ( ( A e. RR /\ B e. RR ) ->
+      A <_ ( ( ( A + B ) + ( abs ` ( A - B ) ) ) / 2 ) ) $=
+    ? $.
+
+  $( The maximum of two reals is no smaller than the first real.
+     Lemma 3.10 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxle1 $p |- ( ( A e. RR /\ B e. RR ) -> A <_ sup ( { A , B } , RR , < ) ) $=
+    ? $.
+
+  $( The maximum of two reals is no smaller than the second real.
+     Lemma 3.10 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxle2 $p |- ( ( A e. RR /\ B e. RR ) -> B <_ sup ( { A , B } , RR , < ) ) $=
+    ? $.
+
+  $( The maximum of two reals is commutative.
+     Lemma 3.9 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxcom $p |- ( ( A e. RR /\ B e. RR ) ->
+      sup ( { A , B } , RR , < ) = sup ( { B , A } , RR , < ) ) $=
+    ? $.
+
+  $( Equivalence of ` <_ ` and being equal to the maximum of two reals.
+     Lemma 3.12 of [Geuvers], p. 10.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  maxleb $p |- ( ( A e. RR /\ B e. RR ) ->
+      ( A <_ B <-> sup ( { A , B } , RR , < ) = B ) ) $=
+    ? $.
+
+  $( Absolute value defined in terms of maximum.
+     Definition 3.13 of [Geuvers], p. 11.
+     (Contributed by Jim Kingdon, 21-Dec-2021.) $)
+  dfabsmax $p |- ( A e. RR -> ( abs ` A ) = sup ( { A , -u A } , RR , < ) ) $=
+    ? $.
```

Comment:
The inequality >= is an easy consequence of leabs and absneg (by the way, -A <= |A| may be worth stating). The inequality <= is a consequence of 0 <= max(A, -A). For the latter, do you have in mind specific results already formalized for sup and/or Dedekind cuts that migh apply ?

Reply:
> The inequality >= is an easy consequence of leabs and absneg . . .
> The inequality <= . . . [follows from] ~absle.

This formalizes nicely. I added you to the Contributed line because the above is a sketch of the proof I ended up formalizing.